### PR TITLE
💬1081-modify-pause-notification-banner-text-to-be-less-ambiguous

### PR DESCRIPTION
**Pull request by jonhindle**

### CODE REVIEW

File: server/helpers/utilities.js [hunk 1]
```diff
@@ -110,7 +110,7 @@ const hmppsAuthMFAUser = (token) => jwtDecode(token).authorities.includes('ROLE_
 
 const getSnoozeUntil = (rawSnoozeUntil) => {
   const snoozeUntil = moment(rawSnoozeUntil)
-  return snoozeUntil.isAfter(moment()) ? snoozeUntil.format('dddd, Do MMMM, YYYY') : ''
+  return snoozeUntil.isAfter(moment()) ? snoozeUntil.add(1, 'day').format('dddd, Do MMMM YYYY') : ''
```
Comment (astroash) [inline server/helpers/utilities.js:113]: Do you want `moment().endOf('day')` here?

File: server/views/partials/snooze-banner.ejs [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
-  <div class="govuk-grid-column-two-thirds">
+  <div class="govuk-grid-column-three-quarters">
 
     <svg class="moj-banner__icon" fill="currentColor" role="presentation" focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 25 25" height="25" width="25">
       <path d="M13.7,18.5h-2.4v-2.4h2.4V18.5z M12.5,13.7c-0.7,0-1.2-0.5-1.2-1.2V7.7c0-0.7,0.5-1.2,1.2-1.2s1.2,0.5,1.2,1.2v4.8C13.7,13.2,13.2,13.7,12.5,13.7z M12.5,0.5c-6.6,0-12,5.4-12,12s5.4,12,12,12s12-5.4,12-12S19.1,0.5,12.5,0.5z" />
     </svg>
 
     <div class="moj-banner__message">
-      <h2 class="govuk-heading-s govuk-!-padding-bottom-2">Notifications paused until <%= snoozeUntil %></h2>
+      <h2 class="govuk-heading-s govuk-!-padding-bottom-2">Your notifications are currently turned off. You have set them to start again on <%= snoozeUntil %>. Or, you can choose to resume them at any time.</h2>
```
Comment (astroash) [inline server/views/partials/snooze-banner.ejs:9]: Well, this is going to break my integration tests 😂

File: server/helpers/utilities.js (hunk 1 above)
Comment (nickmcmahon01) [reply server/helpers/utilities.js:113]: It either needs to say 'Until 23:59:59 Monday' or 'Until Tuesday' IMO

Comment (jonhindle) [reply server/helpers/utilities.js:113]: Not in this case. This represents a date to be displayed when the notifications will resume. 
`EndOf` would return the end of the day albeit by a second, but the format applied would still display the previous day and not the next one. Or at least thats my understanding.

Comment (jonhindle) [reply server/helpers/utilities.js:113]: I wrote that then panicked and went off to check; https://momentjs.com/docs/#/manipulating/end-of/
My reading of that, is it still returns the previous day 🤔

File: server/views/partials/snooze-banner.ejs (hunk 2 above)
Comment (jonhindle) [reply server/views/partials/snooze-banner.ejs:9]: Sorry. Our _copy_ chap did a thorough job on it though 👍

**Pull request merged**